docs/manual-setup.adoc

```diff
@@ -145,12 +145,97 @@ const tbtc = await TBTC.withConfig({
 })
 ```
 
+and add
+```
+import { BitcoinHelpers } from "@keep-network/tbtc.js"
+```
+to the import section if it isn't there.
```

Comment:
This can actually move to editing the config.json file now, I think 🤔

Reply:
Actually no, I think this whole section is no longer necessary, as the network id 1101 is already pointed to regtest in tbtc-dapp's config: https://github.com/keep-network/tbtc-dapp/blob/master/src/config/config.json